app/bundles/CoreBundle/Service/FlashBag.php

```diff
@@ -46,7 +45,10 @@ public function add($message, $messageVars = [], $level = self::LEVEL_NOTICE, $d
             $translatedMessage = $this->translator->trans($message, $messageVars, $domain);
         }
 
-        $this->session->getFlashBag()->add($level, $translatedMessage);
+        $session = $this->requestStack->getSession();
+        if ($session instanceof Session) {
```

Comment:
I'd suggest changing this to an assert, or a better to an exception if one is calling this service not within the general symfony request, notifying the user of the code, that this must be called only with `Session`.

Reply:
Do you have more details about how to do this when the code is run via CLI? An assert won't help in this case.

Reply:
Umm, why flashbag should run in a CLI?

Reply:
No, but the code that is using it is being called from CLI as well.

Reply:
Well.. This will need a refactoring session. But it's ok to have it as you have.